moebot_bot/util/db/scheduledOperation.go

```diff
@@ -0,0 +1,116 @@
+package db
+
+import (
+	"log"
+	"time"
+)
+
+type SchedulerType int
+
+const (
+	SchedulerChannelRotation SchedulerType = iota
```

Comment:
For anything serialized to the database we should be using explicit numbers instead of relying upon `iota`. That way sorting won't break existing rows in the DB.

I know some old db code does `iota`, but for new code we should make sure to follow this pattern.